### PR TITLE
Template/real result

### PR DESCRIPTION
**Pull request by rizkiaaprilian**
real result template, yotpo, testimonials carousel & filter function

### CODE REVIEW

File: src/scss/sections/_testimonials-carousel.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.testimonials-carousel {
+  background-color: $gray-100;
```
Comment (foria) [inline src/scss/sections/_testimonials-carousel.scss:2]: please let's use bg class for this, if there is not, let's create it in utilities

File: src/scss/components/_carousel.scss [hunk 2]
```diff
@@ -156,6 +156,12 @@
 		}
 	}
 
+  &--body {
```
Comment (foria) [inline src/scss/components/_carousel.scss:159]: why do we need of it?

File: src/partials/sections/testimonials-carousel.hbs [hunk 3]
```diff
@@ -0,0 +1,80 @@
+<section class="testimonials-carousel mt-4 py-4">
+  <div class="container text-center">
+    <h2 class="h1 mb-0">As seen in</h2>
+    <div id="testimonialsCarousel" class="carousel slide" data-ride="carousel">
+
+      <div class="carousel-inner">
+        <div class="carousel-item active">
```
Comment (foria) [inline src/partials/sections/testimonials-carousel.hbs:7]: quote icons as on LIVE are missing, please integrate them by using the SVG version (ask to designers the SVG)

File: src/docs/templates/real-result.hbs [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+
+<head>
+	{{> html-head title="Real Result" }}
+</head>
+
+<body>
+  {{> sections/header}}
+  <section class="py-4 real-result">
+    <div class="container">
+      <h1 class="text-center">Real Customers. Real Reviews</h1>
```
Comment (foria) [inline src/docs/templates/real-result.hbs:12]: use a bigger margin-bottom as on LIVE

**Review by foria — CHANGES_REQUESTED** [review]

File: src/scss/components/_carousel.scss (hunk 2 above)
Comment (rizkiaaprilian) [reply src/scss/components/_carousel.scss]: I changed to black variant, its needed to set indicator active color

File: src/scss/sections/_testimonials-carousel.scss [hunk 5]
```diff
@@ -1,3 +1,33 @@
 .testimonials-carousel {
-  background-color: $gray-100;
-}
+    &__quote::before {
```
Comment (foria) [inline src/scss/sections/_testimonials-carousel.scss:2]: please don't duplicate properties, let's have first quote:before, quote:after together with the common style and then each of them with their single styles

**Review by foria — CHANGES_REQUESTED** [review]

**Review by foria — APPROVED** [review]

**Pull request merged**